examples/ddpg_pmsm_dq_current_control.py

```diff
@@ -20,83 +20,138 @@
 from gym.core import Wrapper
 from gym.spaces import Box, Tuple
 
+# This example shows how we can use GEM to train a reinforcement learning agent to control the current within
+# a permanent magnet synchronous motor three-phase drive
+# We use a deep-deterministic-policy-gradient (DDPG) agent
+# to determine which action must be taken on a continuous-control-set
+
 
 class AppendLastActionWrapper(Wrapper):
     """
     The following environment considers the dead time in the real-world motor control systems.
-    The real-world system changes its state, while the agent calculates the next action based on a previoulsly measured
-    observation. Therefore, for the agents it seems as if the applied action effects the state one step delayed.
-    (with a dead time of one time-step)
+    The real-world system changes its state, while the agent simultaneously calculates the next action based on a
+    previously measured observation.
+    Therefore, for the agents it seems as if the applied action affects the environment with one step delay
+    (with a dead time of one time step).
 
-    For complete observability of the system at each time-step we append the last played action of the agent to the
-    observation, because this action will be the one that is active in the next step.
+    For complete observability of the system at each time step we append the last selected action to the

```

Comment:
The state space is completly observable within this PMSM example even without the action information. A delay is just a delay and nothing which is shadowing parts of the state space. Hence, appending the last action(s) is not related to a POMDP-style framework but just simple feature eng.